components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/APIProviderImpl.java

```diff
@@ -5227,6 +5228,48 @@ public boolean isConfigured() {
         return certificateManager.getCertificates(tenantId);
     }
 
+    @Override
+    public List<CertificateMetadataDTO> searchCertificates(int tenantId, String alias, String endpoint) throws
+            APIManagementException {
+        CertificateManager certificateManager = new CertificateManagerImpl();
```

Comment:
In other methods we have a space after the method definition and in this one we do not? What is the correctly formatted one? Shall we be consistent and stick to one pattern?